### PR TITLE
Improved lyrics

### PR DESCRIPTION
**Pull request by mattpike11**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,43 +1,30 @@
...
-I am the one and only
-Nobody I'd rather be
-I am the one and only
-You can't take that away from me
 
 I am the one and only
 Nobody I'd rather be
 I am the one and only
 You can't take that away from us
+>>>>>>> main
```
Comment (adrianclay) [inline README.md:30]: This looks like a merge marker that has been left in?

Comment (tom-clarke-dluhc) [inline README.md:30]: VS code messed up...

File: README.md [hunk 2]
```diff
@@ -1,43 +1,30 @@
...
-We are the two and together
+Matt is the one and only
 Nobody I'd rather be
-I am the one and only
-You can't take that away from me
-
+Matt is the one and only
+You can't take that away from him
 I can't wear this uniform without some compromises
+But I can wear a bib (unlike Mario)
```
Comment (kateriley1) [inline README.md:23]:
```suggestion
But I can wear a bib (unlike Mario!)
```

File: README.md [hunk 3]
```diff
@@ -1,43 +1,30 @@
-# The One and Only
+# merge-conflict-demo
```
Comment (adrianclay) [inline README.md:1]: I preferred the old title.


Perhaps with an updated twist?
```suggestion
# The One and Only Matt
```

File: README.md [hunk 4]
```diff
@@ -1,43 +1,30 @@
...
-
+Nobody I'd rather be
+Matt is the one and only
+You can't take that away from him
 I've been a player in the crowd scene
-A dot on the big screen
+A flicker on the big screen
 My soul embraces one more in a million faces
 High hopes and aspirations, and years above my station
 Maybe but all this time I've tried to walk with dignity and pride
```
Comment (tom-clarke-dluhc) [inline README.md]:
```suggestion
Maybe but all this time I've tried to walk without dignity and pride
```

File: README.md [hunk 5]
```diff
@@ -1,43 +1,30 @@
...
-You can't take that away from me
-
+Nobody I'd rather be
+Matt is the one and only
+You can't take that away from him
 I've been a player in the crowd scene
-A dot on the big screen
+A flicker on the big screen
 My soul embraces one more in a million faces
 High hopes and aspirations, and years above my station
```
Comment (tom-clarke-dluhc) [inline README.md]:
```suggestion
High hopes and perspirations, and years above my station
```

**Review by tom-clarke-dluhc — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (mattpike11) [reply README.md:30]: ok

File: README.md (hunk 2 above)
Comment (mattpike11) [reply README.md:23]: nice one

File: README.md (hunk 3 above)
Comment (mattpike11) [reply README.md:1]: sweet

File: README.md (hunk 1 above)
Comment (mattpike11) [reply README.md:30]: sweet

File: README.md (hunk 5 above)
Comment (mattpike11) [reply README.md]: noice

**Pull request merged**